### PR TITLE
feat(classroom): add user guide to the classroom

### PR DESCRIPTION
**Pull request by Cheerego7**

### CODE REVIEW

File: desktop/renderer-app/src/stores/class-room-store.ts [hunk 1]
```diff
@@ -443,6 +450,22 @@ export class ClassRoomStore {
         this.addMessage(RTMessageType.ChannelMessage, text, this.userUUID);
     };
 
+    public onUserGuide = (): void => {
+        if (
+            this.messages.length > 0 &&
+            this.messages[this.messages.length - 1].type === RTMessageType.UserGuide
```
Comment (BlackHole1) [inline desktop/renderer-app/src/stores/class-room-store.ts:461]: why? add comment

File: web/flat-web/src/stores/class-room-store.ts [hunk 2]
```diff
@@ -435,6 +442,22 @@ export class ClassRoomStore {
         this.addMessage(RTMessageType.ChannelMessage, text, this.userUUID);
     };
 
+    public onUserGuide = (): void => {
+        if (
+            this.messages.length > 0 &&
+            this.messages[this.messages.length - 1].type === RTMessageType.UserGuide
```
Comment (BlackHole1) [inline web/flat-web/src/stores/class-room-store.ts:453]: add comment

**Review by BlackHole1 — CHANGES_REQUESTED** [review]

File: desktop/renderer-app/src/stores/class-room-store.ts (hunk 1 above)
Comment (BlackHole1) [reply desktop/renderer-app/src/stores/class-room-store.ts:461]: Why not use: `this.message.some(({ type }) => type === RTMessageType.UserGuide)`

Why do I need to take the last digit of the array?(`this.messages[this.messages.length - 1]`)

Comment (Cheerego7) [reply desktop/renderer-app/src/stores/class-room-store.ts:461]: when rejoin or refresh classroom that the user guide message always at the end

Comment (BlackHole1) [reply desktop/renderer-app/src/stores/class-room-store.ts:461]: Ok. Then it would be better to add this sentence to the comment.

**Review by BlackHole1 — APPROVED** [review]
LGTM

**Pull request merged**